src/app/views/staff/staff.controller.coffee

```diff
@@ -19,10 +20,8 @@
         _.find(vm.subTenants, (subTenant) -> subTenant.id == vm.staff.mnoe_sub_tenant_id).name
       vm.staff.adminRoleName = ->
         _.find(vm.adminRoles, (role) -> role.value == vm.staff.admin_role).label
-
-      if(vm.staff.mnoe_sub_tenant_id)
-        MnoeSubTenants.get(vm.staff.mnoe_sub_tenant_id).then((r) -> vm.staff.subTenants = r.data.sub_tenants)
   )
+  # Temporary solution, does not scale if there is more than 50 subtenants
   MnoeSubTenants.list(null, null, null).then((response) -> vm.subTenants = response.data)
```

Comment:
Why can't this be paginated?

Reply:
Because it's too fill a Combo Box.

Reply:
Then the solution would be to reuse `angucomplete-alt` lib. Already used in this project.